src/System.Collections.NonGeneric/src/System/Collections/Hashtable.cs

```diff
@@ -736,11 +733,7 @@ protected virtual bool KeyEquals(Object item, Object key)
         // 
         public virtual ICollection Keys
         {
-            get
-            {
-                if (_keys == null) _keys = new KeyCollection(this);
-                return _keys;
```

Comment:
Please revert this change but do feel free to put this if block on its own line to make it match the coding guidelines.